python/ray/workflow/tests/test_lifetime.py

```diff
@@ -36,8 +36,8 @@ def test_workflow_lifetime_1(workflow_start_cluster):
     # Case 1: driver exits normally
     address, storage_uri = workflow_start_cluster
     with patch.dict(os.environ, {"RAY_ADDRESS": address}):
-        ray.init(storage=storage_uri)
-        run_string_as_driver(driver_script.format(storage_uri, 5))
+        ray.init()
```

Comment:
I think we need to call:
```
ray.init(address=address)
```

to connect to the existing cluster.

Reply:
@iycheng the `RAY_ADDRESS` env is set on the line above.  I also missed that line at first when I updated these tests.

Reply:
Got it.